feconf.py

```diff
@@ -569,6 +569,8 @@ def get_empty_ratings():
 NEW_EXPLORATION_URL = '/contributehandler/create_new'
 PREFERENCES_DATA_URL = '/preferenceshandler/data'
 QUESTION_DATA_URL = '/questionhandler'
+QUESTION_MANAGER_URL = '/questionmanagerhandler'
+QUESTION_POST_URL = '/questionposthandler'
```

Comment:
Let's call this QUESTION_CREATION_URL = '/questioncreationhandler'. We've never added "post" or "get" to the URL before.

Reply:
updated.